astro/src/content/quickstarts/quickstart-javascript-nextjs-web.mdx

```diff
@@ -0,0 +1,277 @@
+
+## Getting Started
+In this section, you'll get FusionAuth up and running, and configured with the ChangeBank application.
+
+### Clone The Code
+First off, grab the code from the repository and change into that directory.
+
+```
+git clone https://github.com/FusionAuth/fusionauth-quickstart-javascript-nextjs-web.git
+cd fusionauth-quickstart-javascript-nextjs-web
+```
+
+### Run FusionAuth Via Docker
+
+In the root directory of the repo you'll find a Docker compose file (docker-compose.yml) and an environment
+variables configuration file (.env). Assuming you have Docker installed on your machine, you can stand up
+FusionAuth up on your machine with:
+
+```
+docker compose up -d
+```
+
+This will start three containers, once each for FusionAuth, Postgres, and Elastic.
+
+Here you are using a bootstrapping feature of FusionAuth, called [Kickstart](/docs/v1/tech/installation-guide/kickstart).
+When FusionAuth comes up for the first time, it will look at the `kickstart/kickstart.json` file and
+configure FusionAuth to a certain initial state.
+
+<Aside type="note">
+If you ever want to reset the FusionAuth system, delete the volumes created by docker compose by
+executing `docker compose down -v`, then re-run `docker compose up -d`.
+</Aside>
+
+FusionAuth will be configured with these settings
+* Your client Id is `c2aeeb1e-65cd-43c5-9326-02dce5e03705`.
+* Your client secret is `2HYT86lWSAntc-mvtHLX5XXEpk9ThcqZb4YEh65CLjA-not-for-prod`.
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth `http://localhost:9011/`.
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don’t need to.
+
+### Full Example
+
+Make sure once again that you are in the directory `complete-application/`.
+
+Copy the `.env.example` to `.env.local`
+
+```shell
+cp .env.example .env.local
+```
+
+The environment variables will match what is in the `/kickstart/kickstart.json` file that is used when first starting FusionAuth. If you have changed anything, like creating a new Application you will need to update these environment variables accordingly.
+
+`.env.local`
+<RemoteCode url={frontmatter.coderoot + "/complete-application/.env.example"}
+  lang="ts" />
+
+Install required packages
+
+```shell
+npm install
+```
+
+### Walkthrough Example
+
+In this section, you'll set up a basic Next.js application with two pages. 
+
+1. Homepage
+1. Account - protected
+
+#### Setup Your Environment
+
+Create a new application using the `npx`.
+
+```shell
+npx create-next-app@latest changebank
+```
+
+Accept all of the default prompts like below
+
+![npx installation defaults](/img/articles/quickstart-javascript-nextjs-web/npx-create-next-app.png)
```

Comment:
Please put in the terminal output here.

Reply:
Also would be awesome if we could specify all of these commands on the command line:

`npx create-next-app@latest changebank --no-tailwind ...`

Is that possible?